lib/metrics/RedisClient.js

```diff
@@ -22,6 +22,16 @@ class RedisClient {
         return this;
     }
 
+    /**
+    * get keys for given pattern
+    * @param {string} pattern - string pattern to match with all existing keys
+    * @param {callback} cb - callback (error, result)
+    * @return {undefined}
+    */
+    keys(pattern, cb) {
+        return this._client.keys(pattern, cb);
```

Comment:
Instead of `keys`, I would recommend using `scan` https://redis.io/commands/scan since `keys` can accidentally cause a performance hit on the system.
From the docs

> Warning: consider KEYS as a command that should only be used in production environments with extreme care. It may ruin performance when it is executed against large databases. This command is intended for debugging and special operations, such as changing your keyspace layout. Don't use KEYS in your regular application code. If you're looking for a way to find keys in a subset of your keyspace, consider using SCAN or sets.